plugins/catalog-backend-module-gitlab/src/providers/GitlabOrgDiscoveryEntityProvider.ts

```diff
@@ -208,6 +359,7 @@ export class GitlabOrgDiscoveryEntityProvider implements EntityProvider {
       groups = paginated<GitLabGroup>(options => client.listGroups(options), {
         page: 1,
         per_page: 100,
+        all_available: true,
```

Comment:
Will `all_available: true` change how the provider behaves for existing adopters? If so, perhaps we could make this change optional through some configuration?

Reply:
Hi @vinzscam ,

so, there is a nice discussion going on about it [here](https://github.com/backstage/backstage/pull/23287). My colleague @markussiebert sent a PR that would allow the parameters for this [api call](https://docs.gitlab.com/ee/api/groups.html#list-groups) to be configurable. However, as @freben said, it would be probably a burden to the majority of the users to have this many options... 

At this time I cannot think of a better solution...I believe having this hardcoded would be benefitial to all users right now as it would allow them to list every group their token has access to, no matter if the token is linked to a regular user or admin. What do you think?

Reply:
I still think every option that allows filter the results at gitlab api level isn't a bad option ;-)

Reply:
thank you both, I've missed the discussion. Sorry for causing some confusion 😂
Yeah if `all_available: true` is going to "always be set" then I agree, let's keep it hardcoded.

> I still think every option that allows filter the results at gitlab api level isn't a bad option ;-)

that's absolutely true, I agree! But sometimes it's nice to have less code to maintain, so that we only maintain the configurations that matter.